src/screens/AddNewItemScreen/index.tsx

```diff
@@ -36,12 +40,36 @@ export const AddNewItemScreen: FC = () => {
 
   return (
     <View style={style.container}>
-      <TextInput
-        style={style.input}
-        placeholder="New task"
-        onChangeText={setText}
-        value={text}
-        selectionColor={COLORS.black}
+      <View style={{ width: '100%' }}>
+        <TouchableOpacity
+          style={style.input}
+          onPress={() => inputRef.current?.focus()}>
+          <Text>{'Add new task'}</Text>
+          <TextInput
+            ref={inputRef}
+            placeholder="New task"
+            onChangeText={setText}
+            value={text}
+            selectionColor={COLORS.black}
+          />
+        </TouchableOpacity>
+        <TouchableOpacity onPress={() => setOpen(true)} style={style.input}>
+          <Text>{'Set the reminder send time'}</Text>
+          <Text>{`${currentDate}`}</Text>
+        </TouchableOpacity>
+      </View>
+      <DatePicker
+        minimumDate={new Date()}
+        modal
+        open={open}
+        date={date}
+        onConfirm={date => {
```

Comment:
create functions before return (for each arrow function that you created in tsx)

Reply:
ok